### PR TITLE
dbeaver/pro#2049 Notation switch doesn't work in a custom ER Diagram

### PR DESCRIPTION
**Pull request by serjiokov**
Fixed notation containment for custom diagram.

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/editor/ERDEditorStandalone.java [hunk 1]
```diff
@@ -201,7 +202,11 @@ private EntityDiagram loadContentFromFile(DBRProgressMonitor progressMonitor)
         IStorage storage = EditorUtils.getStorageFromInput(getEditorInput());
 
         final DiagramPart diagramPart = getDiagramPart();
-
+        ERDNotationDescriptor diagramNotation=null;
```
Comment (E1izabeth) [inline plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/editor/ERDEditorStandalone.java]:
```suggestion
        ERDNotationDescriptor diagramNotation = null;
```

Comment (E1izabeth) [reply plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/editor/ERDEditorStandalone.java]: 🙃

**Review by E1izabeth — APPROVED** [review]

File: plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/editor/ERDEditorStandalone.java (hunk 1 above)
Comment (serjiokov) [reply plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/editor/ERDEditorStandalone.java]: Done

**Review by E1izabeth — APPROVED** [review]

**Review by ShadelessFox — APPROVED** [review]

**Comment by MashaKorax** [comment]
verified

**Pull request merged**